server/routes/notification.js

```diff
@@ -0,0 +1,167 @@
+const authMiddleware = require("../middlewares/authMiddleware");
+const ListProductModel = require("../models/listProductModel");
+const NotificationModel = require("../models/notificationModel");
+const UserListModel = require("../models/userListModel");
+const { PRICE } = require("../utils/enums");
+const router = require("express").Router();
+
+//Make the given notification id read
+router.put("/read/:id", authMiddleware, async (req, res) => {
+  try {
+    const { id } = req.params;
+    const { id: user_id } = req.user;
+
+    await NotificationModel.update(
+      { isRead: true },
+      {
+        where: {
+          id,
+          user_id,
+        },
+      }
+    );
+
+    //update returns only a 1 or a 0 so we create another query to show the client
+    const newNotification = await NotificationModel.findOne({
```

Comment:
Will this information be more useful than just the ID of the notification?

Reply:
I'm not sure what youre asking, but if youre asking about the returned value, I returned the notification with the updated isRead property. I figured returning the updated model would be better than just the default success value of the Model.update which is [1] .